src/OrchardCore.Modules/OrchardCore.Flows/Extensions/FlowMetadataExtemtions.cs

```diff
@@ -0,0 +1,39 @@
+using System;
+using System.Collections.Generic;
+using System.Text;
+using OrchardCore.Flows.Models;
+
+namespace OrchardCore.Flows.Extensions
+{
+    public static class FlowMetadataExtemtions
```

Comment:
Do we really require this? What about other grid systems?

Reply:
@hishamco requested to move it to function

Reply:
agreed. will remove

Reply:
If I recall I just need you to move into a function inside `@functions` block inside razor view

Reply:
Deleted the unnecessary extension file